boards/librem13v2/librem13v2.config

```diff
@@ -34,3 +34,4 @@ export CONFIG_BOOT_GUI_MENU_NAME="Purism Librem 13v2 Heads Boot Menu"
 export CONFIG_USB_BOOT_DEV="/dev/sdb1"
 export CONFIG_WARNING_BG_COLOR="--background-gradient 0 0 0 150 125 0"
 export CONFIG_ERROR_BG_COLOR="--background-gradient 0 0 0 150 0 0"
+export CONFIG_BOARD="librem"
```

Comment:
Maybe the `CONFIG_BOARD` variable should be injected into `/etc/config` the same way `GIT_HASH` 
 and `GIT_STATUS` are in the main Makefile?

Reply:
On one hand that would remove an area of redundancy (such as if it reused the BOARD environment variable). In the Librem case it would mean that we would need to define each board revision within the flash.sh script instead of having a more generic label as we have boards that require different coreboot configuration but would be able to use the same flashrom options. I suppose your suggestion would make it more consistent with how we have the board configurations organized thus far. I'm open to either approach.

Reply:
I'd defer to @osresearch. I just figured it would be one less thing to worry about in board-specific config files having `BOARD` be injected automatically.  And to handle multiple versions with single logic, you could just change to use `librem*)` in the case statement in `flash.sh` (for as long as that holds true for all revisions of your boards).

Reply:
The more I think about it, the more I like the idea of reusing a configuration option we already have in place. It makes things more consistent and it ties directly into the idea of options for a particular board. I will take that approach.

Reply:
Makefile change incorporated.